test/linalg/lu.jl

```diff
@@ -0,0 +1 @@
+@test logdet(Complex64[1.0f0 0.5f0; 0.5f0 -1.0f0]) === 0.22314355f0 + 3.1415927f0im
```

Comment:
If you want to move chunks of other pre-existing tests into this file, sure that makes sense. A 1-line file is pretty pointless and not worth the noise.